dp/cloud/python/magma/db_service/tests/alembic_testcase.py

```diff
@@ -62,10 +62,15 @@ def get_table(self, table_name: str) -> sa.Table:
         return sa.Table(table_name, sa.MetaData(), autoload_with=self.engine)
 
     def has_column(self, table: sa.Table, column_name: str) -> bool:
-        for c in table.columns:
-            if c.name == column_name:
-                return True
-        return False
+        return self.has_columns(table=table, columns_names=[column_name])
+
+    def has_columns(self, table: sa.Table, columns_names: List[str]) -> bool:
+        existing = [c.name for c in table.columns]
```

Comment:
I'd rather make this a set instead of list for faster lookup

also I would suggest either adding a todo or adjusting existing tests that use has_column in a for loop to reduce the complexity